docs/api.rst

```diff
@@ -0,0 +1,249 @@
+.. _ksql-http-api:
+
+HTTP API Reference
+==================
+
+Content Types
+-------------
+
+The KSQL HTTP server uses content types for requests and responses to indicate the serialization format of the data. Currently, the only serialization format supported is JSON, specified as ``application/json``. Your request should specify this content type in the ``Accept`` header::
+
+    Accept: application/json
+
+Errors
+------
+
+All API endpoints use a standard error message format for any requests that return an HTTP status indicating an error (any 4xx or 5xx statuses):
+
+   .. sourcecode:: http
```

Comment:
this code sample tag shouldn't be indented. if so, it displays like this (extra vertical bar):
![image](https://user-images.githubusercontent.com/11722533/39443948-e60229a2-4c6a-11e8-890c-54696a0357b2.png)

should be simply:

```
.. sourcecode:: http

      HTTP/1.1 <Error Status>
      Content-Type: application/json

      {
          "message": <Error Message>
      }

```